pkg/build/builder/common.go

```diff
@@ -20,7 +22,14 @@ import (
 // client facing libraries should not be using glog
 var glog = utilglog.ToFile(os.Stderr, 2)
 
-const OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
+const (
+	OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
```

Comment:
@bparees please open a follow up to move this to pkg/build/api and add Godoc.  It doesn't follow our API conventions either.